includes/validation/class-amp-invalid-url-post-type.php

```diff
@@ -198,21 +200,30 @@ public static function get_invalid_url_validation_errors( $post, $args = array()
 				continue;
 			}
 			$term = get_term_by( 'slug', $stored_validation_error['term_slug'], AMP_Validation_Error_Taxonomy::TAXONOMY_SLUG );
-			if ( ! $term ) {
-				continue;
+
+			$forced_sanitization = null;
+			if ( $args['check_forced_sanitization'] ) {
+				$error = $stored_validation_error['data'];
+				unset( $error['sources'] );
+
+				/** This filter is documented in amp/includes/validation/class-amp-validation-manager.php */
+				$forced_sanitization = apply_filters( 'amp_validation_error_sanitized', $forced_sanitization, $error );
 			}
-			if ( $args['ignore_accepted'] && AMP_Validation_Error_Taxonomy::VALIDATION_ERROR_ACCEPTED_STATUS === $term->term_group ) {
+
+			if ( $args['ignore_accepted'] && ( true === $forced_sanitization || ( $term && AMP_Validation_Error_Taxonomy::VALIDATION_ERROR_ACCEPTED_STATUS === $term->term_group ) ) ) {
 				continue;
 			}
-			$errors[] = array(
+			if ( $term && null !== $forced_sanitization ) {
```

Comment:
Yes. `true` means that the validation error is forced to be accepted (sanitized), whereas`false` means that the validation error is forced to be rejected (not sanitized). A `null` value means that there is no forcing because no filter is applying to force the sanitized state.